openmetadata-ui/src/main/resources/ui/playwright/e2e/Flow/ApiServiceRest.spec.ts

```diff
@@ -105,7 +107,7 @@ test.describe('API service', () => {
 
     await toastNotification(
       page,
-      /deleted successfully!/,
+      /(deleted successfully!|Delete operation initiated)/,
       BIG_ENTITY_DELETE_TIMEOUT
     );
```

Comment:
@dhruvjsx what's the reason for this change? we are indeed trying to confirm if the delete action is successful or not, we don't want to just confirm if it's triggered or not.